block/manager.go

```diff
@@ -161,6 +164,11 @@ func NewManager(
 		return nil, fmt.Errorf("get initial state: %w", err)
 	}
 
+	err = m.LoadDrsHistoryFromStore()
```

Comment:
as to my other comment - don't think maintining this in memory is necessary

Reply:
removed

Reply:
drs history replaced by drs per height in store